server/http/protolet/protolet.go

```diff
@@ -9,10 +9,16 @@ import (
 	"net/http"
 	"reflect"
 
+	ctxpb "github.com/buildbuddy-io/buildbuddy/proto/context"
+  "github.com/buildbuddy-io/buildbuddy/server/util/request_context"
```

Comment:
needs auto formatting, can you run ./buildfix.sh or whatever build hooks you prefer?